Allow setting the value/unit color in a ListQuantity

ybott-qinetic opened the pull request:
Fixes #2438

mr-manuel commented:
This sets the color of the quantity value. The color of the quantity unit needs also to be fixed.

ybott-qinetic commented:
@mr-manuel thank you, patch updated

mr-manuel reviewed (approved):
Looks good now!

blammit commented:
@ybott-qinetic could you update the message to explain this this being done in order to align the colours with those in ListQuantityGroup?

ybott-qinetic committed:
Allow setting the value/unit color in a ListQuantity

This patch aligns the theme colors in ListQuantity with those
in ListQuantityGroup.

Fixes #2438
